int32 joint
int32 zeroOffsets
---
bool success


